src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/service/ActionService.java

```diff
@@ -142,7 +145,7 @@ public void saveFlowLatency(String flowId, String direction, long latency) {
     public void sendFlowSyncRequest(String flowId) {
         Flow flow = flowRepository.findById(flowId)
                 .orElseThrow(() -> new IllegalStateException(format("Flow %s not found.", flowId)));
-        if (LATENCY_BASED_STRATEGIES.contains(flow.getPathComputationStrategy())) {
+        if (LATENCY_BASED_STRATEGIES.contains(flow.getPathComputationStrategy()) && isReactionsEnabled()) {
```

Comment:
You still write in a log notification about future reaction, despite value of feature toggle.

Reply:
Fixed.